### PR TITLE
Add support for Angular 2 HTML

### PR DESCRIPTION
**Pull request by stevium**
Register `Angular2HtmlFocusedElementFinder` to support `Angular2HtmlLanguage` and handle the parents as `HtmlTag` PsiElements.

image

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Review by mallowigi — APPROVED** [review]

File: build.gradle.kts [hunk 1]
```diff
@@ -64,6 +65,7 @@ intellij {
     // javascript
     "JavaScript",
     "Pythonid:$pluginsVersion",
+    "AngularJS:$angularVersion",
```
Comment (mallowigi) [inline build.gradle.kts:69]: You need to add this value to the gradle.properties, if it's not the same as $pluginsVersion

**Comment by mallowigi** [comment]
Looks like you didn't add the version to the gradle.properties

File: build.gradle.kts (hunk 1 above)
Comment (stevium) [reply build.gradle.kts:69]: Added it now, but please note that locally I was only able to test with following setup: 

```gradle
pluginsVersion=233.9102.97
sassVersion=233.9102.195
angularVersion=233.9102.195
...
platformVersion=233.9102.97-EAP-SNAPSHOT
```

**Comment by stevium** [comment]
> Looks like you didn't add the version to the gradle.properties

Yes sorry, forgot to push it, but had it locally. 

Nevertheless, I think the current master build is not compatible with `LATEST-EAP-SNAPSHOT`. So I think it would be good to consolidate the versions if possible.

I was not able to find a working `sass/python` Version. because Intellij is ahead of it: `IU-233.9802.14`: 

image

**Pull request merged**